miqa/core/models/scan.py

```diff
@@ -5,6 +5,8 @@
 from django.db import models
 from django_extensions.db.models import TimeStampedModel
 
+SCAN_TYPES = [('ncanda-t1spgr-v1', 'ncanda-t1spgr-v1'), ('ncanda-mprage-v1', 'ncanda-mprage-v1')]
```

Comment:
`ncanda-mprage-v1` is repeated twice. Is this intentional?

Reply:
Yes, when specifying `choices` for a django `CharField`, the items in the list must be tuples of `(label, value)`. In many cases, it is good practice to have the label and value the same.